AdminUi/src/AdminUi/appsettings.json

```diff
@@ -1,5 +1,6 @@
 {
     "AllowedHosts": "*",
+    "Urls": "http://*:5000",
```

Comment:
This should be set via `ASPNETCORE_URLS` environment variable instead. Also, use `0.0.00` instead of `*`. Not sure if it's better, but at least it's consistent with the consumer api then. :)